### PR TITLE
[artifactory] 7.6.2 release

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:
[artifactory] 7.6.2 release

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -743,14 +745,14 @@ database:
...
-  preStartCommand: "mkdir -p /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ."  
+  preStartCommand: "mkdir -p /opt/center/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/center/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ."
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:748]: I think this was accidentally changed. all directory paths went from `/opt/jfrog` to `/opt/center/jfrog`

File: stable/artifactory-ha/README.md [hunk 2]
```diff
@@ -1032,7 +1034,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `logger.image.tag`        | tag for logger image              | `1.30`                                                   |
 | `artifactory.name`                   | Artifactory name                     | `artifactory`                              |
 | `artifactory.image.pullPolicy`       | Container pull policy                | `IfNotPresent`                             |
-| `artifactory.image.repository`       | Container image                      | `docker.bintray.io/jfrog/artifactory-pro`  |
+| `artifactory.image.repository`       | Container image                      | `docker.bintray.io/center/jfrog/artifactory-pro`  |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1037]: Was this changed on purpose?

File: stable/artifactory-ha/README.md [hunk 3]
```diff
@@ -1107,7 +1109,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.haBackupDir.enabled`         | Enable haBackupDir for eventual storage in the HA cluster   | `false` |
 | `artifactory.haBackupDir.path`         | Path to the directory intended for use with NFS eventual configuration for HA   | |
 | `artifactory.migration.timeoutSeconds`          | Artifactory migration Maximum Time out in seconds| `3600`       |
-| `artifactory.persistence.mountPath`    | Artifactory persistence volume mount path           | `"/var/opt/jfrog/artifactory"`  |
+| `artifactory.persistence.mountPath`    | Artifactory persistence volume mount path           | `"/var/opt/center/jfrog/artifactory"`  |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1112]: Same here - looks like a mistake

File: stable/artifactory-ha/README.md [hunk 4]
```diff
@@ -1121,8 +1123,8 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.persistence.nfs.ip`            | NFS server IP                        |                                     |
 | `artifactory.persistence.nfs.haDataMount`   | NFS data directory                   | `/data`                             |
 | `artifactory.persistence.nfs.haBackupMount` | NFS backup directory                 | `/backup`                           |
-| `artifactory.persistence.nfs.dataDir`       | HA data directory                    | `/var/opt/jfrog/artifactory-ha`     |
-| `artifactory.persistence.nfs.backupDir`     | HA backup directory                  | `/var/opt/jfrog/artifactory-backup` |
+| `artifactory.persistence.nfs.dataDir`       | HA data directory                    | `/var/opt/center/jfrog/artifactory-ha`     |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1126]: Same, probably a mistake

File: stable/artifactory-ha/README.md [hunk 5]
```diff
@@ -1121,8 +1123,8 @@ The following table lists the configurable parameters of the artifactory chart a
...
+| `artifactory.persistence.nfs.backupDir`     | HA backup directory                  | `/var/opt/center/jfrog/artifactory-backup` |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1127]: Again, probably a mistake

File: stable/artifactory-ha/README.md [hunk 6]
```diff
@@ -1167,8 +1169,8 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.persistence.azureBlob.containerName`   | Azure Blob Storage container name      | ``                        |
 | `artifactory.persistence.azureBlob.testConnection`  | Azure Blob Storage test connection     | `false`                   |
 | `artifactory.persistence.fileSystem.existingSharedClaim`  | Enable using an existing shared pvc | `false`                             |
-| `artifactory.persistence.fileStorage.dataDir`             | HA data directory                   | `/var/opt/jfrog/artifactory/artifactory-data`     |
-| `artifactory.persistence.fileStorage.backupDir`           | HA backup directory                 | `/var/opt/jfrog/artifactory-backup` |
+| `artifactory.persistence.fileStorage.dataDir`             | HA data directory                   | `/var/opt/center/jfrog/artifactory/artifactory-data`     |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1172]: Same

File: stable/artifactory-ha/README.md [hunk 7]
```diff
@@ -1167,8 +1169,8 @@ The following table lists the configurable parameters of the artifactory chart a
...
+| `artifactory.persistence.fileStorage.backupDir`           | HA backup directory                 | `/var/opt/center/jfrog/artifactory-backup` |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1173]: Same

File: stable/artifactory-ha/README.md [hunk 8]
```diff
@@ -1242,7 +1244,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.replicaCount`        | Nginx replica count               | `1`                                                    |
 | `nginx.uid`                 | Nginx User Id                     | `104`                                                  |
 | `nginx.gid`                 | Nginx Group Id                    | `107`                                                  |
-| `nginx.image.repository`    | Container image                   | `docker.bintray.io/jfrog/nginx-artifactory-pro`        |
+| `nginx.image.repository`    | Container image                   | `docker.bintray.io/center/jfrog/nginx-artifactory-pro`        |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1247]: Probably a mistake again

File: stable/artifactory-ha/README.md [hunk 9]
```diff
@@ -1297,7 +1299,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.resources.requests.cpu`    | Nginx initial cpu request                  | `100m`                                  |
 | `nginx.resources.limits.memory`   | Nginx memory limit                         | `250Mi`                                 |
 | `nginx.resources.limits.cpu`      | Nginx cpu limit                            | `500m`                                  |
-| `nginx.persistence.mountPath` | Nginx persistence volume mount path | `"/var/opt/jfrog/nginx"`                           |
+| `nginx.persistence.mountPath` | Nginx persistence volume mount path | `"/var/opt/center/jfrog/nginx"`                           |
```
Comment (danielezer) [inline stable/artifactory-ha/README.md:1302]: Same

File: README.md [hunk 10]
```diff
@@ -10,10 +10,12 @@ Get the latest [Helm release](https://github.com/kubernetes/helm#install).
...
 
-You need to add this Chart repo to Helm:
+### Add ChartCenter Helm repository
 
-```console
-helm repo add jfrog https://charts.jfrog.io/
+Before installing JFrog helm charts, you need to add the [ChartCenter helm repository](https://chartcenter.io) to your helm client
+
+```bash
+helm repo add center https://repo.chartcenter.io
```
Comment (danielezer) [inline README.md:18]: Is `center` really the best name? I would use `chartcenter` instead.

File: stable/artifactory/README.md [hunk 11]
```diff
@@ -606,7 +609,7 @@ This can be done with the following parameters
 # Make sure your Artifactory Docker image has the MySQL database driver in it
 ...
 --set postgresql.enabled=false \
---set artifactory.preStartCommand="wget -O /opt/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.1.41.jar https://jcenter.bintray.com/mysql/mysql-connector-java/5.1.41/mysql-connector-java-5.1.41.jar" \
+--set artifactory.preStartCommand="wget -O /opt/center/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.1.41.jar https://jcenter.bintray.com/mysql/mysql-connector-java/5.1.41/mysql-connector-java-5.1.41.jar" \
```
Comment (danielezer) [inline stable/artifactory/README.md:612]: Probably a mistake again

File: stable/artifactory/README.md [hunk 12]
```diff
@@ -628,14 +631,14 @@ database:
...
-  preStartCommand: "mkdir -p /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ."  
+  preStartCommand: "mkdir -p /opt/center/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib; cd /opt/center/jfrog/artifactory/var/bootstrap/artifactory/tomcat/lib && wget -O instantclient-basic-linux.x64-19.6.0.0.0dbru.zip https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && unzip -jn instantclient-basic-linux.x64-19.6.0.0.0dbru.zip && wget -O libaio1_0.3.110-3_amd64.deb http://ftp.br.debian.org/debian/pool/main/liba/libaio/libaio1_0.3.110-3_amd64.deb &&  dpkg-deb -x libaio1_0.3.110-3_amd64.deb . && cp lib/x86_64-linux-gnu/* ."
```
Comment (danielezer) [inline stable/artifactory/README.md:634]: Same here - probably a mistake

File: stable/artifactory/README.md [hunk 13]
```diff
@@ -772,9 +775,9 @@ You can configure the sidecar to run as a custom user if needed by setting the f
 ```
 
 ### Add Artifactory User Plugin during installation
-If you need to add [Artifactory User Plugin](https://github.com/jfrog/artifactory-user-plugins), you can use this option.
+If you need to add [Artifactory User Plugin](https://github.com/center/jfrog/artifactory-user-plugins), you can use this option.
```
Comment (danielezer) [inline stable/artifactory/README.md:778]: Same, mistake

File: stable/artifactory/README.md [hunk 14]
```diff
@@ -772,9 +775,9 @@ You can configure the sidecar to run as a custom user if needed by setting the f
...
 
-Create a secret with [Artifactory User Plugin](https://github.com/jfrog/artifactory-user-plugins) by following command:
+Create a secret with [Artifactory User Plugin](https://github.com/center/jfrog/artifactory-user-plugins) by following command:
```
Comment (danielezer) [inline stable/artifactory/README.md:780]: Same

File: stable/artifactory/README.md [hunk 15]
```diff
@@ -803,7 +806,7 @@ artifactory: # Name of the artifactory dependency
     userPluginSecrets:
       - '{{ template "my-chart.fullname" . }}'
 ```
-NOTE: By defining userPluginSecrets, this overrides any pre-defined plugins from the container image that are stored in /tmp/plugins.  At this time [artifactory-pro:6.9.0](https://bintray.com/jfrog/artifactory-pro) is distributed with `internalUser.groovy` plugin.  If you need this plugin in addition to your user plugins, you should include these additional plugins as part of your userPluginSecrets.
+NOTE: By defining userPluginSecrets, this overrides any pre-defined plugins from the container image that are stored in /tmp/plugins.  At this time [artifactory-pro:6.9.0](https://bintray.com/center/jfrog/artifactory-pro) is distributed with `internalUser.groovy` plugin.  If you need this plugin in addition to your user plugins, you should include these additional plugins as part of your userPluginSecrets.
```
Comment (danielezer) [inline stable/artifactory/README.md:809]: Same

File: stable/artifactory/README.md [hunk 16]
```diff
@@ -910,7 +913,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.name`        | Artifactory name                  | `artifactory`                                            |
 | `artifactory.replicaCount`            | Replica count for Artifactory deployment| `1`                                    |
 | `artifactory.image.pullPolicy`        | Container pull policy             | `IfNotPresent`                               |
-| `artifactory.image.repository`        | Container image                   | `docker.bintray.io/jfrog/artifactory-pro`    |
+| `artifactory.image.repository`        | Container image                   | `docker.bintray.io/center/jfrog/artifactory-pro`    |
```
Comment (danielezer) [inline stable/artifactory/README.md:916]: Same

File: stable/artifactory/README.md [hunk 17]
```diff
@@ -977,7 +980,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.database.maxOpenConnections`         | Maximum amount of open connections from Artifactory to the DB   | `80` |
 | `artifactory.copyOnEveryStartup`         | List of files to copy on startup from source (which is absolute) to target (which is relative to ARTIFACTORY_HOME   |  |
 | `artifactory.migration.timeoutSeconds`          | Artifactory migration Maximum Timeout in seconds| `3600`       |
-| `artifactory.persistence.mountPath`      | Artifactory persistence volume mount path        | `"/var/opt/jfrog/artifactory"`       |
+| `artifactory.persistence.mountPath`      | Artifactory persistence volume mount path        | `"/var/opt/center/jfrog/artifactory"`       |
```
Comment (danielezer) [inline stable/artifactory/README.md:983]: Same, mistake

File: stable/artifactory/README.md [hunk 18]
```diff
@@ -992,8 +995,8 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.persistence.nfs.ip`            | NFS server IP                        |                                     |
 | `artifactory.persistence.nfs.haDataMount`   | NFS data directory                   | `/data`                             |
 | `artifactory.persistence.nfs.haBackupMount` | NFS backup directory                 | `/backup`                           |
-| `artifactory.persistence.nfs.dataDir`       | HA data directory                    | `/var/opt/jfrog/artifactory`     |
-| `artifactory.persistence.nfs.backupDir`     | HA backup directory                  | `/var/opt/jfrog/artifactory-backup` |
+| `artifactory.persistence.nfs.dataDir`       | HA data directory                    | `/var/opt/center/jfrog/artifactory`     |
```
Comment (danielezer) [inline stable/artifactory/README.md:998]: Same, mistake

File: stable/artifactory/README.md [hunk 19]
```diff
@@ -992,8 +995,8 @@ The following table lists the configurable parameters of the artifactory chart a
...
+| `artifactory.persistence.nfs.backupDir`     | HA backup directory                  | `/var/opt/center/jfrog/artifactory-backup` |
```
Comment (danielezer) [inline stable/artifactory/README.md:999]: Same, mistake

File: stable/artifactory/README.md [hunk 20]
```diff
@@ -1081,7 +1084,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.replicaCount` | Nginx replica count | `1`                                                                         |
 | `nginx.uid`                 | Nginx User Id                     | `104`                                                  |
 | `nginx.gid`                 | Nginx Group Id                    | `107`                                                  |
-| `nginx.image.repository`    | Container image                   | `docker.bintray.io/jfrog/nginx-artifactory-pro`        |
+| `nginx.image.repository`    | Container image                   | `docker.bintray.io/center/jfrog/nginx-artifactory-pro`        |
```
Comment (danielezer) [inline stable/artifactory/README.md:1087]: Same mistake

File: stable/artifactory/README.md [hunk 21]
```diff
@@ -1128,7 +1131,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `nginx.tlsSecretName` |  SSL secret that will be used by the Nginx pod |                                                 |
 | `nginx.customConfigMap`           | Nginx CustomeConfigMap name for `nginx.conf` | ` `                                   |
 | `nginx.customArtifactoryConfigMap`| Nginx CustomeConfigMap name for `artifactory.conf` | ` `                             |
-| `nginx.persistence.mountPath` | Nginx persistence volume mount path | `"/var/opt/jfrog/nginx"`                           |
+| `nginx.persistence.mountPath` | Nginx persistence volume mount path | `"/var/opt/center/jfrog/nginx"`                           |
```
Comment (danielezer) [inline stable/artifactory/README.md:1134]: Again, probably the same mistake

File: README.md (hunk 10 above)
Comment (chukka) [reply README.md:18]: yes

**Pull request merged**